### ISSUE TITLE
comment-whitespace not working correctly?

### ISSUE THREAD

**Issue by RohanNagar** [sleekbyte/tailor#214]
When I run Tailor on my `.swift` file that looks like this:

```swift
  /// Retreives a user.
  ///
  /// - note: The network call is made asynchronously.
  ///
  /// - parameters:
  ///    - username: The name to retrieve user information for.
  ///
  func getUser(username: String) {
```

I get the following output, where line 33 is the beginning of that comment block and line 39 is the end of the comment block:

```
/Users/rohannagar/Documents/Developer/UserService.swift:33:5: warning: [comment-whitespace] Single-line comment should start with whitespace
/Users/rohannagar/Documents/Developer/UserService.swift:34:5: warning: [comment-whitespace] Single-line comment should start with whitespace
/Users/rohannagar/Documents/Developer/UserService.swift:35:5: warning: [comment-whitespace] Single-line comment should start with whitespace
/Users/rohannagar/Documents/Developer/UserService.swift:36:5: warning: [comment-whitespace] Single-line comment should start with whitespace
/Users/rohannagar/Documents/Developer/UserService.swift:37:5: warning: [comment-whitespace] Single-line comment should start with whitespace
/Users/rohannagar/Documents/Developer/UserService.swift:38:5: warning: [comment-whitespace] Single-line comment should start with whitespace
/Users/rohannagar/Documents/Developer/UserService.swift:39:5: warning: [comment-whitespace] Single-line comment should start with whitespace
```

There are two problems with this: 1, there is in fact whitespace after each single-line comment that has text following it, so I'm not sure why I'm getting the warnings. And two, documentation comment lines without text should not require a whitespace after `///`. 

I am getting this very often on all of my `.swift` files. Let me know if you need more information or if I am doing something completely wrong. Thanks!

**Comment by adityatrivedi** [comment]
Hey @RohanNagar, it is a bug and has been fixed in #211.  The bugfix will be incorporated the upcoming release very soon.

**Comment by RohanNagar** [comment]
Ah, apologies. Thanks for the quick reply.
